src/components/badges/Badge.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { FlexWrapper, Image, Typography } from "components";
+
+interface Styles {
+    backgroundColor?: string;
```

Comment:
please, rethink type. if you pass the type of your theme Colors, than you wouldn't be able to add any invalid color